v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/SpannerToSourceDb.java

```diff
@@ -356,6 +356,17 @@ public interface Options extends PipelineOptions, StreamingOptions {
     Integer getDlqRetryMinutes();
 
     void setDlqRetryMinutes(Integer value);
+
+    @TemplateParameter.Enum(
+        order = 24,
+        optional = true,
+        description = "Source database type, ex: mysql",
+        enumOptions = {@TemplateEnumOption("mysql")},
+        helpText = "The type of source database to reverse replicate to.")
+    @Default.String("mysql")
```

Comment:
Is this advisable? This is a decision we will not be able to easily walk back...
What is the harm is this being a mandatory parameter that the user has to configure?